src/sage/topology/simplicial_set_examples.py

```diff
@@ -790,3 +794,65 @@ def HopfMap():
     return S3.Hom(S2)({alpha_1:s0_sigma, alpha_2:s1_sigma,
                        alpha_3:s2_sigma, alpha_4:s0_sigma,
                        alpha_5:s2_sigma, alpha_6:s1_sigma})
+
+
+def PresentationComplex(G):
+    r"""
+    Return a simplicial set constructed from a group presentation.
+    The result is a subdivision of the presentation complex.
+
+    The presentation complex has one vertex and one edge for
```

Comment:
```suggestion
    The presentation complex has a single vertex and it has one edge for
```
(It is possible to read the current version as meaning one vertex for each generator and one edge for each generator.)